src/vs/code/electron-main/windows.ts

```diff
@@ -90,6 +90,12 @@ interface INativeOpenDialogOptions {
 	window?: VSCodeWindow;
 }
 
+interface IConfiguration {
```

Comment:
@xwvvvvwx I suggest to move this into `window.ts` including the configuration change listener. I think it is OK to let each window handle this change from within instead of from outside.